library/src/main/java/com/novoda/downloadmanager/DownloadMigratorBuilder.java

```diff
@@ -0,0 +1,45 @@
+package com.novoda.downloadmanager;
+
+import android.content.Context;
+import android.os.Handler;
+import android.os.Looper;
+
+public class DownloadMigratorBuilder {
```

Comment:
This is one of the two classes that the client now interacts with. `DownloadMigrator` being the other.